docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/cards/block-content--cgov-external-link-block--embedded-feature-card.html.twig

```diff
@@ -8,10 +8,16 @@
 
 {# ####### Set Image ######## #}
 {% if content.field_override_image_promotional|field_value %}
-  {% set image = content.field_override_image_promotional|field_value %}
+	{% set image = content.field_override_image_promotional|field_value %}
 {% endif %}
 
 {# ###### Setup URL ######## #}
 {% set url = content.field_featured_url[0]['#url'] %}
 
-{{ cardMacros.featureCard(title, description, image, url) }}
+{% set id = node.id()|clean_unique_id %}
```

Comment:
This is not correct. You are not a node entity, but you are a block_content entity. This should be throwing errors...